enclave-runtime/src/state.rs

```diff
@@ -72,19 +72,24 @@ impl HandleState for StateFacade {
 	}
 }
 
-pub fn load_initialized_state(shard: &H256) -> SgxResult<StfState> {
+/// Load the state for a given shard
+///
+/// Initializes the shard and state if necessary, so this is guaranteed to
+/// return a state
+pub fn load_initialized_state(shard: &ShardIdentifier) -> Result<StfState> {
```

Comment:
Could we rename this function? For me this name suggests that it loads an already initialized state. So maybe it should rather be  something like `load_or_initialize_state` ?

Reply:
Well it always loads a state. In case none exists, it initializes and loads it